.github/workflows/daily.yml

```diff
@@ -87,10 +88,10 @@ jobs:
             if [ "${{ runner.os }}" = "Windows" ] && [ -n "$PACKAGES" ]; then
               choco install -y $PACKAGES
             fi
-
-            python tests/stubtest_third_party.py --specified-stubs-only --num-shards 4 --shard-index ${{ matrix.shard-index }}
           fi
 
+          $PYTHON_EXECUTABLE tests/stubtest_third_party.py --specified-stubs-only --num-shards 4 --shard-index ${{ matrix.shard-index }}
```

Comment:
@AlexWaygood is this code de-duplication worth making a PR for? (adds a line, but makes the command to run have a single source of truth)

Reply:
Sure, seems like a nice improvement :)

I'd vote for this control flow, though:

```bash
if [ "${{ runner.os }}" = "Linux" ]; then
  PYTHON_EXECUTABLE="xvfb-run python"
else
  PYTHON_EXECUTABLE="python"
fi
```

rather than

```bash
PYTHON_EXECUTABLE="python"

if [ "${{ runner.os }}" = "Linux" ]; then
  PYTHON_EXECUTABLE="xvfb-run python"
fi
```

I find the former much more readable.